ddd-kata.md

```diff
@@ -0,0 +1,197 @@
+
+1. You can enter and leave the parking lot or the parking garage through several entrances and exits.
+2. There are specific parking spots for different types of vehicles (motorcycles, car, electric car, truck/bus, handicapped persons, family-friendly parking spots)
+3. Each parking spot has a unique identifier (for parking garage – number & floor or for parking lot – number &  area)
+4. The customer may collect a ticket at each entrance and leave via any exit.
+5. The ticket given to customer contains a parking spot ID where to park
+6. The ticketing information must be stored for 10 years due to tax reasons.
+7. There are three types of terminals:
+   1. The terminals at the entrances have a way for the customer to communicate the vehicle details to the system.
+   2. Customer drive into the parking garage / lot via entrance terminals where the collect the tickets
+   3. The terminals near the exits have a way to pay the ticket.
+   4. Detail out the actual payment process like select payment method, put in credit card, enter PIN are not relevant here.
+8. The terminals at the exit gates collect the paid ticket.
+9. The ticket has a read/write magnetic stripe containing information including a position to park.
+10. The magnetic stripe can be changed by the terminals.
+11. Parking garages have a parking guiding system, that guides customers to the correct parking spot. The guiding system recognizes the vehicle and guides it to the right parking spot.
+12. Parking lots do not have such a system.
+
+As domain expert, please stay from requirements perspective within these guardrails. You can detail them out as much as you want.
+
+### The Process
+
+#### Define the Team
+
+![Team Setup](./images/team-setup.png)
+
+As a **Moderator / Facilitator**, your task is to organize the workshops and conduct the workshops.
+
+As a **Domain Expert**, your tasks are:
+
+* Mimics the expert for parking lot software & management
+* Takes business decisions, has the last say on requirements
+* Researches and creates requirements
+
+As a **Team Member** your tasks are:
+
+* Interview the Domain Expert
+* Create the deliverables
+* Participate actively in all workshops
+
+#### Plan your timeline
+
+![Timeline Proposal](./images/timeline-proposal.png)
+
+* Plan to have ~4h a week of modelling sessions
+* Work iteratively (e.g. while working on Aggregate Canvas you might want to adjust your Event Storming)
+* As a moderator, strife for consensus and alignment in the team
+* Plan your time carefully. Timebox your efforts.
+* Most importantly, have fun modelling.
+
+#### How to run an Event Storming Workshop
+
+Event storming is a collaborative workshop method that helps to explore complex business domains and model them using events. An event is an action that occurred in the system at a specific time. Event storming can be conducted in two phases: a high-level event storm to identify the domains and then a detailed event storm to focus on a core domain.
```

Comment:
```suggestion
Event Storming is a collaborative workshop method that helps to explore complex business domains and model them using events. An event is an action that occurred in the domain at a specific time. Event Storming can be conducted in two phases: a high-level Event Storming to identify the domains and then a detailed Event Storming to focus on a specific subdomain.
```

Reply:
We should find a consistent way to spell "Event Storming". In the table above it's spelled "EventStorming" (i.e. without a space).